### ISSUE TITLE
CloseWindow already defined (again)

### ISSUE THREAD

**Issue by lab313ru** [raysan5/raylib#3848]
Hi,

I saw two closed issues related to the problem, but it's still here.

I have to use other headers in my x64 builds, and also I'm using vcpkg, so I can't change libs linking order or rename. How can I fix this compilation error then?

**Comment by raysan5** [comment]
Including the libraries with duplicate symbols in separate modules or changing the linkage order is the proper solution.

**Comment by lab313ru** [comment]
But vcpkg doesn't allow you to change this order.

**Comment by raysan5** [comment]
@lab313ru I'm afraid I don't understand how vcpkg is involved in the build process of a custom project, please, could you provide more details? How are you building your project and raylib?

**Comment by lab313ru** [comment]
First, I've installed raylib and raygui:
```bash=
vcpkg install raylib:x64-windows-static raygui:x64-windows-static
```

After that I've included raylib.h, and compiled my code. Visual Studio automatically links required libs for that, so you don't have to specify it in the libs list. 

P. S. I've fixed the error: specifying raylib.lib manually as the first lib anyway (even if VS does link it automatically) makes it appeared twice in the linked libs list, but it fixes the linkage order and allows to compile your code. I would recommend it as a required solution in readmes for compiling in Windows.
